### DISCUSSION TITLE
Deploying sftpGO on Synology NAS

### DISCUSSION THREAD

**Discussion by MirDima** [drakkan/sftpgo#730] [Q&A] [answered]
Deploying sftpGO on Synology NAS
Hi! Thank you for your project and what you are doing for us!!
I have a problem setting up sftpgo in a Docker container on Synology. When installing the container, the Poker application asks for a path, I specify a local folder on the server, but I don't see the path on WebAdmin. by default, it is different. How do I specify the path to the local folder on the server?
Thanks!
![Screenshot 2022-02-21 at 23 52 04](https://user-images.githubusercontent.com/92049077/155025305-28167e12-c380-4237-bfcf-0fe89626cdca.png)
![Screenshot 2022-02-21 at 23 51 21](https://user-images.githubusercontent.com/92049077/155025319-d6a6c44e-2f74-4a63-b12c-c89cef0661cd.png)
![Screenshot 2022-02-21 at 23 52 26](https://user-images.githubusercontent.com/92049077/155025331-0039ce1e-5f69-4210-a61d-b52560d8df4a.png)
![Screenshot 2022-02-21 at 23 54 16](https://user-images.githubusercontent.com/92049077/155025342-ea821715-f63a-40e6-93e1-b86638316197.png)

**Comment by drakkan** [comment, marked as answer]
Hi,

I don't have access to a synology NAS to test myself so my answers could be inaccurate. 

Some notes:

1) SFTPGo will create the folder once you login and not when you create the user, so try to login and see if the folder is created
2) I suggest to map the container folder `/srv/sftpgo` to a local folder. SFTPGo will run by default using 1000 as UID/GID, please make sure that this UID can write to the mapped folder otherwise you will get a permission denied error in your logs. You can also try to run as a [different user](https://github.com/drakkan/sftpgo/tree/main/docker#running-as-an-arbitrary-user) but I don't know how to configure this using the NAS UI

Maybe [this](https://www.wundertech.net/how-to-use-docker-on-a-synology-nas/) could help

**Reply by MirDima** [reply]
Thank you!! I'll try to figure it out.

**Reply by drakkan** [reply]
Let me know if you can find a solution, alternatively I can try it myself if I have a remote connection to a synology NAS

**Reply by MirDima** [reply]
Hi! It will be great! I'll set up access to the test NAS server.
THANK YOU!

**Reply by MirDima** [reply]
Could you give me your email please? I'll send credential for NAS Synology.
For now, just check this link http://92.38.195.226:44381/ you could see web interface of my NAS.
Thank you!

**Reply by drakkan** [reply]
> Could you give me your email please? I'll send credential for NAS Synology. For now, just check this link http://92.38.195.226:44381/ you could see web interface of my NAS. Thank you!

You can find my public email on my GitHub [profile](https://github.com/drakkan) or in each commit, thank you

**Reply by drakkan** [reply]
This was just a permission issue. SFTPGo runs as UID 1000 and you must allow to this UID to write to the mounted volumes by setting the appropriate permissions using the Synology UI
